.github/workflows/_push-entrypoint.yaml

```diff
@@ -245,6 +246,16 @@ jobs:
       builder: ${{ needs.init.outputs.BUILDER }}
       ct-matrix: ${{ needs.prepare.outputs.ct-matrix }}
 
+  update-emqx-docs:
+    if: needs.prepare.outputs.release == 'true'
```

Comment:
Should we also guard that the event type is `tag`?

Reply:
`release` will not be set to `true` if the workflow is not triggered by tag